### ISSUE TITLE
Inverse of artransform

### ISSUE THREAD

**Issue by cristiano1br** [helske/KFAS#15]
I suggest implementing a function to compute the inverse of artransform.

**Comment by helske** [comment]
Yes this might be nice addition, base R has an C function for this [invpartrans](https://github.com/wch/r-source/blob/af7f52f70101960861e5d995d3a4bec010bc89e6/src/library/stats/src/arima.c#L525) and it could be ported easily to Fortran (or called as such). At the moment I do not have time for his though, but pull requests are of course welcome. :)
